docs/docs/index.mdx

```diff
@@ -68,27 +68,30 @@ import Link from "@docusaurus/Link";
         alt="Licence"
         src="https://img.shields.io/github/license/encord-team/encord-active"
       />
+      <img
+        style={{ marginRight: 5 }}
+        alt="Downloads"
+        src="https://static.pepy.tech/badge/encord-active"
+      />
     </td>
   </tr>
 </table>
 
-## ❓ What is Encord Active?
```

Comment:
I don't think that we should remove this title.

Reply:
I removed that line to increase the consistency degree between Encord Annotate and Active docs. Check [annotate](https://docs.encord.com/) against [active](https://docs.encord.com/active/docs). Let me know what you think afterwards.
I prefer to keep that line in the github repo readme but doesn't seem necessary for the docs.